cadquery/hull.py

```diff
@@ -187,6 +189,9 @@ def _pt_arc(p: Point, a: Arc) -> Tuple[float, float, float, float]:
     xc, yc = a.c.x, a.c.y
     dx, dy = x - xc, y - yc
     l = sqrt(dx ** 2 + dy ** 2)
+    ld = l ** 2 - r ** 2
+    if (ld < 0.001):
+        return x, y, x, y
```

Comment:
This is when the point is on the arc's circle, so I return a no-length segment (which is later ignored). The point might not necessarily be on the arc though, so this should return a segment to the start or end of the arc.

This is taken care of elsewhere because I add the start and end points of the arc to the list of points in `convert_and_validate`, so if the next segment is between the point and a start or end point of an arc, it's handled elsewhere as a `pt_pt` comparison.